### DISCUSSION TITLE
so-test does not work

### DISCUSSION THREAD

**Discussion by ivanrdrgz** [Security-Onion-Solutions/securityonion#11850] [Unsupported Versions]
Hello everyone!
I'm using version 2.3.230 (version 2.4 is giving me problems with the hypervisor) and after a standalone installation, I have my network ready on a ESXi Server:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/102434209/d78b3275-ca63-4d0f-ab04-88db978dbaa8)

As you can see in the image, the Security Onion server is connected to a pfSense firewall with two interfaces, the first one 192.168.30.10 (being 192.168.30.1 the firewall gateway) and a sniffing interface connected to a SPAN port, which forwards the traffic of the other three interfaces. I am trying to run so-test command to see if the server gets traffic, but it has been stuck here for over four hours:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/102434209/fa814191-3a56-4112-a84d-79b1b64de546)

Any idea of what is happening? Thank you all in advance!

**Comment by cm-ops** [comment]
`curl -vv ghcr.io/security-onion-solutions/so-tcpreplay` are you able to connect to that?

**Reply by ivanrdrgz** [reply]
Hi, thank you for your answer.

Yes, connection left intact. After a couple hours more, so-test worked. I did absolutly nothing, and I have no idea of why it happened.

So now I would like to see the logs generated by traffic in my network to be sure that Security Onion is working well. If I login to Kibana, I just see logs related to the so-test command. What could be happening here?

Again, thanks a lot for your help.
